coriolisclient/tests/cli/test_utils.py

```diff
@@ -0,0 +1,310 @@
+        )
+
+    def test_get_option_value_from_args_no_value(self):
+        args = mock.Mock()
+        args.option_name = None
+        args.option_name_file = None
+
+        result = utils.get_option_value_from_args(
+            args, "option-name", error_on_no_value=False)
+
+        self.assertEqual(
+            None,
+            result
+        )
+
+        self.assertRaises(
+            ValueError,
+            utils.get_option_value_from_args,
+            args,
+            "option-name"
+        )
+
+    def test_get_option_value_from_args_json_value_error(self):
+        args = mock.Mock()
+        args.option_name = "invalid"
+        args.option_name_file = None
+
+        self.assertRaises(
+            ValueError,
+            utils.get_option_value_from_args,
+            args,
+            "option-name"
+        )
+
+    @ddt.data(
+        {
+            "global_scripts": None,
+            "instance_scripts": None,
+            "expected_result":
+            {
+                "global": {},
+                "instances": {}
+            }
+        },
+        {
+            "global_scripts": ["linux="],
+            "instance_scripts": ["instance_1="],
+            "expected_result":
+            {
+                "global": {"linux": None},
+                "instances": {"instance_1": None}
+            }
+        },
+        {
+            "global_scripts": ["linux script"],
+            "instance_scripts": ["linux script"],
+            "expected_result":
+            {
+                "global": {},
+                "instances": {}
+            }
+        },
+        {
+            "global_scripts": ["invalid_os=scrips"],
+            "instance_scripts": None,
+            "expected_result": None
+        },
+        {
+            "global_scripts": ["linux='invalid/file/path'"],
+            "instance_scripts": None,
+            "expected_result": None
+        },
+        {
+            "global_scripts": None,
+            "instance_scripts": ["linux='invalid/file/path'"],
+            "expected_result": None
+        },
+    )
+    def test_compose_user_scripts(self, data):
+        global_scripts = data["global_scripts"]
+        instance_scripts = data["instance_scripts"]
+        expected_result = data["expected_result"]
+
+        if expected_result:
+            result = utils.compose_user_scripts(
+                global_scripts, instance_scripts)
+
+            self.assertEqual(
+                expected_result,
+                result
+            )
+        else:
+            self.assertRaises(
+                ValueError,
+                utils.compose_user_scripts,
+                global_scripts,
+                instance_scripts
+            )
+
+    def test_compose_user_scripts_from_file(self):
+        script_path = os.path.dirname(os.path.realpath(__file__))
+        script_path = os.path.join(script_path, 'data')
+        script_path = os.path.join(script_path, 'user_scripts.yml')
```

Comment:
You can simplify this by:
```suggestion
        script_path = os.path.join(script_path, 'data/user_scripts.yml')
```